tests/experimental/agent_loop/test_basic_agent_loop.py

```diff
@@ -27,9 +27,29 @@
 from verl.utils import hf_tokenizer
 
 
+def init_fsdp_config() -> DictConfig:
+    config = OmegaConf.load("verl/trainer/config/ppo_trainer.yaml")
+    # test sleep/wake_up with fsdp offload
+    config.actor_rollout_ref.actor.fsdp_config.param_offload = True
+    config.actor_rollout_ref.actor.fsdp_config.optimizer_offload = True
+    return config
+
+
+def init_megatron_config() -> DictConfig:
+    config = OmegaConf.load("verl/trainer/config/ppo_megatron_trainer.yaml")
+    config.actor_rollout_ref.actor.megatron.tensor_model_parallel_size = 2
+    config.actor_rollout_ref.actor.megatron.pipeline_model_parallel_size = 2
+
+    # FIXME: sglang with megatron param_offload got error:
+    # "CUDA error: an illegal memory access was encountered"
```

Comment:
@chenhaiq sglang seems don't work with megatron param_offload=True

Reply:
Can you test with this change?
https://github.com/volcengine/verl/pull/2365/files#diff-3ea40b07302076f020ceb946048c8129b9bd9141feebef80312da55c431de3feR155

Looks like we have some duplicated work in:
https://github.com/volcengine/verl/pull/2365/files#diff-336120079b31a71a3ec16d4e8a9429f73d5f367d8fa4f2f9a3d89d62a8f5a503R695